tests/integration/test_hot_reload_storage_policy/test.py

```diff
@@ -69,59 +97,88 @@ def started_cluster():
 def set_config(node, config):
     node.replace_config("/etc/clickhouse-server/config.d/config.xml", config)
     node.query("SYSTEM RELOAD CONFIG")
+    time.sleep(1)
```

Comment:
Let's at least put a comment why this sleep is required - to give ClickHouse time to refresh disks